lib/DxilContainer/DxilContainerAssembler.cpp

```diff
@@ -1693,21 +1693,17 @@ void hlsl::SerializeDxilContainerForModule(DxilModule *pModule,
   // Clone module for reflection, strip function defs
   std::unique_ptr<Module> reflectionModule;
   if (bEmitReflection) {
-    if (DXIL::CompareVersions(ValMajor, ValMinor, 1, 5) < 0) {
```

Comment:
Maybe check shader model 6.5 here?

Reply:
For consistency reflection blob will have latest metadata with validator version 0.0 until replaced with new format.